### ISSUE TITLE
When run 'npx prisma db push' or 'npx prisma generate' throw an error: 'default' can't be used as an identifier because it's a keyword

### ISSUE THREAD

**Issue by gzlock** [odroe/prisma-dart#172]
I followed the [tutorial](https://prisma.pub/docs/getting-started.html) to do 

My env
system: Win10
database: PostgreSQL 15
dart: `Dart SDK version: 2.19.4 (stable) (Tue Mar 7 09:58:46 2023 +0000) on "windows_x64"`
orm version: 3.2.0
node.js 18.15.0
prisma(node.js package) ^4.11.0

`schema.prisma` content
```prisma
generator client {
  provider = "dart run orm"
}

datasource db {
  provider = "postgresql"
  url      = "postgresql://xxxxxxx"
}

model User {
  id Int @id @default(autoincrement())
  name String
  email String @unique
}
```

log content
```
> npx prisma db push
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: 
Could not format because the source could not be parsed:

line 29, column 44 of .: 'default' can't be used as an identifier because it's a keyword.
   ╷
29 │ enum QueryMode implements _i1.PrismaEnum { default,
   │                                            ^^^^^^^
   ╵


```

**Comment by gzlock** [comment]
I created a empty project to test and got the same error
https://github.com/gzlock/dart_orm_test

**Comment by medz** [comment]
@gzlock Thx, fixed it in 3.2.1 version

**Comment by medz** [comment]
![IMG_9497](https://user-images.githubusercontent.com/5564821/226186104-e57e6124-f5d0-4e89-8b3b-53b970bf62aa.JPG)
@gzlock 害，原来也是同胞兄弟。来这个 Dart/Flutter 群不？有啥问题还能直接在群里问我。
